sonar-scanner-engine/src/main/java/org/sonar/scanner/sensor/DefaultSensorContext.java

```diff
@@ -156,4 +157,9 @@ public boolean isCancelled() {
   public void addContextProperty(String key, String value) {
     sensorStorage.storeProperty(key, value);
   }
+
+  @Override
+  public void markForPublishing(InputFile inputFile) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1186)